sdk-base/src/main/java/com/mapbox/maps/ExtensionUtils.kt

```diff
@@ -41,14 +41,25 @@ fun MapSnapshotInterface.bitmap(): Bitmap {
 
 /**
  * Extension function to convert [CameraState] to [CameraOptions].
+ *
+ * If [anchor] is specified - [CameraOptions.center] will be explicitly set to NULL
+ * in order for the anchor to apply to the map camera.
```

Comment:
Is this also helpful to users that do not use animation plugin? If this change is only for the animation plugin, shall we make a separate  private/internal  extension function within the animations plugin?

Reply:
There is no sense in setting both `anchor` and `center` at same time (in that case `anchor` is ignored). We did not face such issues before in our gestures because we were actually performing one more redundant initial camera update on `easeTo(duration = 0)`. So this `CameraState(anchor) <-> CameraOptions` should always look like the way it looks like now.